### ISSUE TITLE
UI improvment - close tab button

### ISSUE THREAD

**Issue by risoul** [bwinton/TabCenter#824]
Actualy, the close button appears to the right of tab. The result is when we want to select a tab, we close it. The game is to avoir the close button and move forward to the left before click.
May be putting the close button on the left, before (or over) favicon will sove this.

**Comment by dannyhajj** [comment]
I assume you have the tab center always visible. The change you are proposing is more problematic for others who set their tab center to auto-collapse (like myself).
Since, when it auto-collapses, the only part that shows is the favicon and the first 1 or 2 letters of a page's title.

**Comment by risoul** [comment]
Yes, you are right. With autocolapse there is no problem with close button position.

**Comment by fvsch** [comment]
The proposed solution is not applicable. I can think of one or two other solutions, but they all have their downsides. E.g. if we think the user's intent is to activate the tab, and delay showing the close button and making it active by a little while (500ms, 800ms...), this will get in the way of users who do want to close that tab as quickly as possible.

I'd suggest closing this issue if there is no proposed workable solution.
